examples/subplots_axes_and_figures/demo_tight_layout.py

```diff
@@ -1,7 +1,14 @@
 """
-=================
-Demo Tight Layout
-=================
+===============================
+Resizing axes with tight layout
+===============================
+
+Tight layout (`~.figure.Figure.tight_layout`) attempts to resize subplots in
```

Comment:
Is there any reason not to do simply

~~~
`~.figure.Figure.tight_layout` attempts  to ...
~~~